content/24.vaccines-novel.md

```diff
@@ -3,21 +3,19 @@
 ### Abstract
 
 In the 21^st^ century, several emergent viruses have posed a global threat.
-Each of these pathogens has emphasized the value of rapid and scalable vaccine development programs.
+Each pathogen has emphasized the value of rapid and scalable vaccine development programs.
 The ongoing SARS-CoV-2 pandemic has made the importance of such efforts especially clear.
 
-In particular, new biotechnological advances in vaccine design allowed for new advances in vaccines that provide only the nucleic acid building blocks of an antigen, eliminating many safety concerns.
+New biotechnological advances in vaccinology allow for recent advances that provide only the nucleic acid building blocks of an antigen, eliminating many safety concerns.
 During the COVID-19 pandemic, these DNA and RNA vaccines have facilitated the development and deployment of vaccines at an unprecedented pace.
-This success was attributable at least in part to broader shifts in scientific research relative to prior epidemics.
-For example, the genome of SARS-CoV-2 was available as early as January 2020, allowing for global efforts in the development of DNA and RNA vaccines to begin within two weeks of the international community becoming aware of the new viral threat.
-Additionally, these technologies that were previously only theoretical were found to be not only safe but also to have high vaccine efficacy.
+This success was attributable at least in part to broader shifts in scientific research relative to prior epidemics; the genome of SARS-CoV-2 was available as early as January 2020, facilitating global efforts in the development of DNA and RNA vaccines within two weeks of the international community becoming aware of the new viral threat.
+Additionally, these technologies that were previously only theoretical are not only safe but also to have highly efficacious.
```

Comment:
```suggestion
Additionally, these technologies that were previously only theoretical are not only safe but also highly efficacious.
```
This read awkwardly to me.  Because the journal version has been accepted, can you change this in the proofs?

Reply:
Good news -- it's correct in the proofs! I edited the abstract in the submission browser (because it flagged things it didn't like) and then copied it back into the Word and markdown versions. I did Word correct, but not markdown.